tests/test_geoserver.py

```diff
@@ -2,14 +2,15 @@
 
 import numpy as np
 import pytest
+import ravenpy.utils
```

Comment:
I don't understand what is intended here. This will raise an importerror if the gis stuff is not installed. But just below you're try / excepting the import of gis libraries.

Reply:
I think this was automatically added when I performed a refactor. Thanks for pointing that out.